### ISSUE TITLE
Quest:Hippogryph Master Stephanos

### ISSUE THREAD

**Issue by CrossUCI** [Arkania/ArkCORE-NG#277]
I starting testing draenei, and in the quest Hippogryph Master Stephanos, Stephanos is a draenei hippogryph flight master located at the Seat of the Naaru at the bottom of the East ramp leading down into draenei capital of The Exodar. Stephanos not appear even with gm mode, i looking for Stephanos in world.creature_template and the entry is 17555, i add the entry in the location , i complete the quest but when he give the other quest Quest:Return to Caregiver Chellan to flight to the location from the  Azure Watch you can not flight because is not marked, in the begining i think it was because i recently summon Stephanos so i go to the other location , fly again to the exodor but, still not working, what can i do

**Comment by gpn39f** [comment]
look at forum.. 
i answered in your'e previous thread

**Comment by CrossUCI** [comment]
it's quest 9605
questgiver=16768
questender 17555

our database has wrong 17555 as quest starter to..

both npc are present in database.. both are on phase 169.. so both are always visible...

Did you meen , in quest_template change QuestGiverPortrait to 16768 and QuestTurnInPortrait 17555

**Comment by gpn39f** [comment]
no, it is to delete id 17555, quest 9605 in table creature_queststarter
then the wrong ? is not shown..
then look at youre own phase.. it has to be 169 or 0
then you should see both npc.. if not, try to teleport to them.. some npc are below gound or similar.. 
the guid for teleport you find in table creature

**Comment by CrossUCI** [comment]
I see what happen, the problem is the coordinates of the npc ,the quest said that located at the Seat of the Naaru at the bottom of the East ramp leading down into draenei capital of The Exodar, but when i use .go creature Stephanos , is upstair outside of Seat of the Naaru, it should be here , https://i.ytimg.com/vi/rl_XEKCrrtA/maxresdefault.jpg , but the most important problem, is when yo buy a travel in hypogryph to travel back to the other location, the location not appear to travel, only see the current location 
https://drive.google.com/open?id=1K16wu6QsEe2lNNcZJpeQuL4Cyki2XS_N
https://drive.google.com/open?id=1MIem6Do5oNwhr1iSofHmNljHVpx7rEqo

**Comment by CrossUCI** [comment]
I try it to login in the forum but my password is wrong , i try it to recover but the email dont arrived yet, you say that you have problems with the shoutbox, you can install it again, https://community.mybb.com/mods.php?action=view&pid=385

**Comment by gpn39f** [comment]
mmhh.. i have to check the php send mail..
thanks for youre report..

more report i need : can you add new lines to the shortbox??

**Comment by CrossUCI** [comment]
sorry i can help you, i dont have skill to programming.

**Comment by gpn39f** [comment]
found the missing script.. all is fine now.. :-)

**Comment by CrossUCI** [comment]
Hey frank i see that are new change in the core of github, there is one release update that i can download, i look in the folder of Arkore-NG on google drive but the folder 2018 is empty
